src/reducers/account/index.js

```diff
@@ -20,11 +20,23 @@ const initialState = {
    sentMessage: false
 }
 
-const loaderReducer = (state, { ready }) => {
+const actionsPending = []
+
+const loaderReducer = (state, { type, ready }) => {
    if (typeof ready === 'undefined') {
       return state
    }
-   return { ...state, formLoader: !ready }
+
+   if (!ready) {
+      actionsPending.push(type)
```

Comment:
I think there might be problems with mutating this collection directly:
https://redux.js.org/faq/immutable-data/#why-is-immutability-required-by-redux
That might also explain why I had weirdness in Redux dev tools (getAccessKeys action not changing stack).

Also you generally want to take current state of the stack from what redux passes as `state`, as otherwise stuff like time travel would be broken.

Reply:
@marcinbodnar do you plan to fix this?

Reply:
@vgrichina Sorry for the delay. Done.